sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/common/TestTextLine.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.io.common;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.apache.beam.sdk.transforms.DoFn;
+
+/**
+ * Used to pass values around within test pipelines for text file based IO.
+ */
+public abstract class TestTextLine {
```

Comment:
<!--new_thread; commit:9c483a5dd68450cfc0d8bf1cfdd151961fa6cdaa; resolved:0-->
I think JDBCIO needed TestRow since that is the type of objects that were generated when reading (a PCollection of TestRow objects). Text I/O doesn't need a similar class since we read a PCollection of Strings. I think it makes more sense to remove this class and move this code into TextIOIT.java.

Reply:
done